enhancements/ingress/set_eip_nlb_ingress.md

```diff
@@ -0,0 +1,1319 @@
+  tlsProfile:
+    ciphers:
+    - ECDHE-ECDSA-AES128-GCM-SHA256
+    - ECDHE-RSA-AES128-GCM-SHA256
+    - ECDHE-ECDSA-AES256-GCM-SHA384
+    - ECDHE-RSA-AES256-GCM-SHA384
+    - ECDHE-ECDSA-CHACHA20-POLY1305
+    - ECDHE-RSA-CHACHA20-POLY1305
+    - DHE-RSA-AES128-GCM-SHA256
+    - DHE-RSA-AES256-GCM-SHA384
+    - TLS_AES_128_GCM_SHA256
+    - TLS_AES_256_GCM_SHA384
+    - TLS_CHACHA20_POLY1305_SHA256
+    minTLSVersion: VersionTLS12
+```
+
+```yaml
+ % oc get svc router-test -n openshift-ingress -o yaml
+ apiVersion: v1
+ kind: Service
+ metadata:
+   annotations:
+     service.beta.kubernetes.io/aws-load-balancer-eip-allocations: eipalloc-0956fea34de4cb7ab,eipalloc-0e9a3077a70de050a,eipalloc-0b69fc4691f54cdd0,eipalloc-01e6ba6cbba1a391b,eipalloc-0e242df173f906112
+     service.beta.kubernetes.io/aws-load-balancer-healthcheck-healthy-threshold: "2"
+     service.beta.kubernetes.io/aws-load-balancer-healthcheck-interval: "10"
+     service.beta.kubernetes.io/aws-load-balancer-healthcheck-timeout: "4"
+     service.beta.kubernetes.io/aws-load-balancer-healthcheck-unhealthy-threshold: "2"
+     service.beta.kubernetes.io/aws-load-balancer-type: nlb
+     traffic-policy.network.alpha.openshift.io/local-with-fallback: ""
+   creationTimestamp: "2024-04-10T00:20:17Z"
+   finalizers:
+     - service.kubernetes.io/load-balancer-cleanup
+   labels:
+     app: router
+     ingresscontroller.operator.openshift.io/owning-ingresscontroller: test
+     router: router-test
+   name: router-test
+   namespace: openshift-ingress
+   ownerReferences:
+     - apiVersion: apps/v1
+       controller: true
+       kind: Deployment
+       name: router-test
+       uid: 466ac236-050b-4583-b2d1-99323f31fcba
+   resourceVersion: "57990"
+   uid: 812fb12c-8932-44bc-87fd-c257b7c0a455
+ spec:
+   allocateLoadBalancerNodePorts: true
+   clusterIP: 172.30.242.110
+   clusterIPs:
+     - 172.30.242.110
+   externalTrafficPolicy: Local
+   healthCheckNodePort: 31251
+   internalTrafficPolicy: Cluster
+   ipFamilies:
+     - IPv4
+   ipFamilyPolicy: SingleStack
+   ports:
+     - name: http
+       nodePort: 30920
+       port: 80
+       protocol: TCP
+       targetPort: http
+     - name: https
+       nodePort: 30575
+       port: 443
+       protocol: TCP
+       targetPort: https
+   selector:
+     ingresscontroller.operator.openshift.io/deployment-ingresscontroller: test
+   sessionAffinity: None
+   type: LoadBalancer
+ status:
+   loadBalancer:
+     ingress:
+       - hostname: a812fb12c893244bc87fdc257b7c0a45-e77e94821922b90d.elb.us-east-1.amazonaws.com
+```
+
+```sh
+ % oc get pods -n openshift-ingress
+NAME                             READY   STATUS    RESTARTS   AGE
+router-default-7b4f96d59-qfj7z   1/1     Running   0          112m
+router-default-7b4f96d59-tthts   1/1     Running   0          112m
+router-test-5bbb685585-2g5f7     1/1     Running   0          7m38s
+router-test-5bbb685585-np9w4     1/1     Running   0          7m38s
+```
```

Comment:
This part is not needed and clutters the instructions.

Reply:
OK removed